platforms/ios/src/TangramMap/TGMapViewController.h

```diff
@@ -267,14 +260,17 @@ NS_ASSUME_NONNULL_END
 - (void)mapView:(nonnull TGMapViewController *)view didCaptureScreenshot:(nonnull UIImage *)screenshot;
 
 /**
- Called whenever scene updates have been applied to the scene file.
- The list of scene update statuses will be emtpy if all updates have been applied successfully.
- Called whenever scene updates have failed to apply to the scene file.
+ If set by the user, this is called after a
+ `-[TGMapViewController loadSceneFile]` or
+ `-[TGMapViewController loadSceneFileAsync:]` or
+ `-[TGMapViewController loadSceneFileAsync:sceneUpdates:]` or
+ `-[TGMapViewController applySceneUpdate]` is completed.
 
  @param mapView a pointer to the map view
- @param sceneUpdateError a NSError containing information about the scene update that failed
+ @param sceneID sceneID corresponding to which this callback will be called 
+ @param sceneError any errors during scene load or update
  */
-- (void)mapView:(nonnull TGMapViewController *)mapView didFailSceneUpdateWithError:(nonnull NSError *)sceneUpdateError;
+- (void)mapView:(nonnull TGMapViewController *)mapView didLoadScene:(int)sceneID withError:(nonnull NSError *)sceneError;
```

Comment:
In fact it looks like we currently _do_ yield a `nil` when a scene loads without error, which seems contrary to the `nonnull` notation.